student_class.rb

```diff
@@ -1,13 +1,17 @@
 require './person_class'
+require_relative 'classroom'
 # Student class
 class Student < Person
-  attr_accessor :classroom
```

Comment:
- [ ] Let's ensure that we can be able to read the `classroom` by adding it to attribute reader. 

`attr_reader :classroom`